Expensify/App
# Utilize reportAction.clientGUID to prevent sequenceNumber mismatch

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/141701

### Tests
1. Upload an attachment
1. Quickly leave a text comment
1. Verify that the text comment appears below the pending attachment
1. Verify that eventually the text comment moves above the pending attachment
1. Verify that one the attachment finishes uploading it will appear below the comment

### Screenshots
![2020-11-02_19-16-08 (1)](https://user-images.githubusercontent.com/32969087/97946741-232a0180-1d40-11eb-93e4-5dfd22a68bdf.gif)
